### DISCUSSION TITLE
Proper mobile layout for monkeytype

### DISCUSSION THREAD

**Discussion by tarushvkodes** [monkeytypegame/monkeytype#2010] [Ideas] [answered]
The current mobile layout on monkeytype is very bad, for it does not auto-scroll, words don't completely show at the edge of the screen, and the words are too small compared to everything else. 

An improvement to the layout would be to make the words fit better on the screen and to accommodate the mobile keyboard better. Also, it would look better if the UI elements were smaller on the top and bottom bar. That way it's similar to the desktop arrangement and size, but on mobile. This could possibly be done if all the nav bar buttons (login, leaderboard, etc.) were consolidated into one menu button.

By making a better monkeytype UI for mobile, users will be able to test their typing speed on a variety of platforms, and it will be more accurate as they do not have to scroll manually to see the rest of the words.

**Comment by Miodec** [comment, marked as answer]
Monkeytype never was a mobile focused website and never will be - it just doesn't make sense. 

Fixing mobile can be done, but its not important.

**Reply by tarushvkodes** [reply]
That's true. Many people still use MonkeyType on mobile to see their speed, as it's different than on PC. A full redesign of the UI would not be necessary, As you said, fixing it should be enough to make it usable. I feel as though if MonkeyType had dynamic sizing, it would work on both mobile and oddly shaped PC screens, improving the look on all platforms.

**Reply by Miodec** [reply]
I mean, it should already kinda work on mobile? Right?

**Reply by tarushvkodes** [reply]
It does, although only in landscape mode. If you put it into portrait mode some words go off the screen and you have to scroll it manually in both orientations.

**Reply by unrealapex** [reply]
Maybe have a different version of the site for mobile?
